lib/media/streaming_engine.js

```diff
@@ -2336,6 +2414,8 @@ shaka.media.StreamingEngine.PlayerInterface;
  *   updating.
  * @property {shaka.net.NetworkingEngine.PendingRequest} operation
  *   Operation with the number of bytes to be downloaded.
+ * @property {shaka.media.SegmentPrefetch} segmentPrefetch
```

Comment:
This should probably be `@property {?shaka.media.SegmentPrefetch}`, to make it clearer that this is expected to sometimes be null.

Reply:
updated